internal/lightcone/harmony/planetaryrendezvous/planetaryrendezvous.go

```diff
@@ -0,0 +1,45 @@
+package planetaryrendezvous
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	dmgBuff key.Modifier = "planetary-rendezvous"
+)
+
+// After entering battle, if an ally deals the same DMG Type as the wearer,
+// DMG dealt increases by 12%.
+
+func init() {
+	lightcone.Register(key.PlanetaryRendezvous, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        4,
+		Path:          model.Path_HARMONY,
+		Promotions:    promotions,
+	})
+	modifier.Register(dmgBuff, modifier.Config{})
```

Comment:
The stacking behavior of this should be `ReplaceBySource` to allow multiple being equipped at once.